01-JS-DOM-APIs/js/jsExercise12.js

```diff
@@ -0,0 +1,30 @@
+function inputTable(){
+  var tableSymbol = [
```

Comment:
This could be:
```
return [
  ['a','b','c' ,'d', 'e'],
  ['f','g','h', 'i', 'k'],
  ['l','m','n', 'o', 'p'],
];
```

And you will earn the variable declaration